lib/css/components/cards.less

```diff
@@ -17,7 +17,7 @@
     }
 
     // CHILD ELEMENTS
-    p:last-of-type {
+    :last-child {
```

Comment:
```suggestion
    > :last-child {
```

Reply:
Ah, good catch! I was mistaken and thought that the `:last-child` _only_ applied to direct descendants.